tests/e2e/test_main.py

```diff
@@ -855,9 +888,8 @@ def test_data_cp_from_cloud_to_storage(
 @pytest.fixture
 def disk(cli_runner: CLIRunner) -> Iterator[str]:
     # Create disk
-    res = cli_runner(["neuro", "disk", "create", "1G"])
+    res = cli_runner(["neuro", "disk", "create", "100M"])
```

Comment:
no need to get 1G. These tests run in parallel, each run will require 1G, so they will conflict.